spec/swagger_helper.rb

```diff
@@ -204,6 +204,16 @@
                          crossposted_at published_at last_comment_at published_timestamp user
                          reading_time_minutes]
           },
+          FollowedTag: {
+            description: "Representation of a followed tag",
+            type: "object",
```

Comment:
(suggestion, non-blocking): 

```suggestion
            type: :object,
```